### DISCUSSION TITLE
Add Support for "accent-color"

### DISCUSSION THREAD

**Discussion by jvanderen1** [tailwindlabs/tailwindcss#5966] [Ideas]
I would like to style checkbox's using the [accent-color](https://developer.mozilla.org/en-US/docs/Web/CSS/accent-color) property. However, this does not currently seem possible in tailwindcss.

**Comment by adamwathan** [comment]
Hey! You'll be pleased to know this is included in v3 which will be out later this month 👍🏻  Can try the alpha now if you like:

https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.0.0-alpha.1

**Reply by jvanderen1** [reply]
Awesome! It's so cool to be able to interact with open source developers. I feel like I'm talking to a CS celebrity 😄
